frmts/vrt/pixelfunctions.cpp

```diff
@@ -2909,6 +3841,118 @@ static CPLErr ExprPixelFunc(void **papoSources, int nSources, void *pData,
         }
     }
 
+#ifdef GDAL_USE_LLVM
+    if (EQUAL(pszDialect, "LLVM"))
```

Comment:
I'm guessing there is a reason this can't be put inside of `gdal::MathExpression::Create()` as another implementation of `gdal::MathExpression` ... maybe add a comment explaining why?